modules/income_and_assets/lib/income_and_assets/pdf_fill/sections/section_10.rb

```diff
@@ -10,30 +10,31 @@ class Section10 < Section
       KEY = {
         # 10a
         'unreportedAsset' => { key: 'F[0].#subform[8].DependentsHaveAssetsNotReported10a[0]' },
+        # 10b-e (only space for four on form)
         'unreportedAssets' => {
           limit: 4,
           first_key: 'otherRelationshipType',
-          # 10b
+          # Q1
           'assetOwnerRelationship' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
           'assetOwnerRelationshipOverflow' => {
             question_num: 10,
-            question_suffix: '(b)',
+            question_suffix: '(1)',
             question_text: "SPECIFY ASSET OWNER'S RELATIONSHIP TO THE VETERAN"
           },
           'otherRelationshipType' => {
             key: "F[0].OtherRelationship10[#{ITERATOR}]",
             question_num: 10,
-            question_suffix: '(b)',
+            question_suffix: '(1)',
             question_text: "SPECIFY ASSET OWNER'S RELATIONSHIP TO THE VETERAN"
           },
-          # 10c
+          # Q2
           'assetType' => {
             key: "F[0].TypeOfAsset10[#{ITERATOR}]",
             question_num: 10,
-            question_suffix: '(d)',
-            question_text: 'SPECIFY TYPE OF ASSET (CASH, ART, ETC.)'
+            question_suffix: '(2)',
+            question_text: 'SPECIFY TYPE OF ASSET (Cash, art, etc.)'
```

Comment:
@jerekshoe Do we think it is weird to have half in all caps and half not?

Reply:
Sorta, but ultimately the extras generator will change the casing anyways. The reason I changed these was to match the exact casing that is used in the question on the PDF